xml/System.Collections.Immutable/ImmutableDictionary`2.xml

```diff
@@ -1571,7 +1572,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the empty instance.</summary>
```

Comment:
That this could be expanded slightly, c.f. [docs for this interface](https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.iimmutabledictionary-2.clear?view=netcore-3.1#System_Collections_Immutable_IImmutableDictionary_2_Clear):

```
Retrieves an empty dictionary that has the same ordering and key/value comparison rules as this dictionary instance.
```

I'd personally also add clarification that `Clear()` has no side-effects on the object itself, despite what its choice of naming implies. Though I'm willing to be convinced that this is pretty evident from both the namespace and the interface name :-)

Reply:
Can you please post your suggested changes as GitHub code suggestions  so I can apply them?

Reply:
```suggestion
        <summary>Retrieves an empty dictionary that has the same ordering and key/value comparison rules as this dictionary instance.</summary>
```